libxrdp/xrdp_bitmap32_compress.c

```diff
@@ -107,7 +107,7 @@ fsplit3(char *in_data, int start_line, int width, int e,
         }
         start_line--;
         cy++;
-        if (out_index > 64 * 64)
+        if (out_index + width + e > 64 * 64)
```

Comment:
I know it's pre-existing code, but if you define a constant for `64 * 64`, it would be easier for me to review the code.